src/Sentry.Unity/UnityLogger.cs

```diff
@@ -5,12 +5,12 @@
 
 namespace Sentry.Unity
 {
-    internal interface IUnityLoggerInterceptor
+    public interface IUnityLoggerInterceptor
```

Comment:
This could be internal and the constructor that takes it too.

Lets add a single public constructor to `UnityLogger` that takes only `SentryOptions` instead of having the one with optional parameter:

```suggestion
    internal interface IUnityLoggerInterceptor
```